governance.md

```diff
@@ -11,6 +11,25 @@ https://github.com/pangeo-data/governance
 
 ### The Steering Council
 
+The Project will have a Steering Council that consists of Project Contributors who have produced contributions that are substantial in quality and quantity, and sustained over at least one year. The overall role of the Council is to ensure, through taking input from the Community, the long-term well-being of the project, both technically and as a community. As Pangeo draws its strength from representing the Community, it strives to avoid fragmentation of the comminity into separate projects, if possible.
+
+During the everyday project activities, council members participate in all discussions, code review and other project activities as peers with all other Contributors and the Community. In these everyday activities, Council Members do not have any special power or privilege through their membership on the Council. However, it is expected that because of the quality and quantity of their contributions and their expert knowledge of the Project Software and Services that Council Members will provide useful guidance, both technical and in terms of project direction, to potentially less experienced contributors.
+
+The Steering Council and its Members play a special role in certain situations. In particular, the Council may:
+
+Make decisions about the overall scope, vision and direction of the project.
+Make decisions about strategic collaborations with other organizations or individuals.
+Make decisions about specific technical issues, features, bugs and pull requests. They are the primary mechanism of guiding the code review process and merging pull requests.
+Make decisions about the Services that are run by The Project and manage those Services for the benefit of the Project and Community.
+
+Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
+
+Outcomes and decisions should be clearly communicated to the community. This can take three different forms: 
+
+* New challenges for Pangeo will be detailed as a github issue for the community to discuss.
+* Resolved technical deadlocks will result in an explanation on the relevant issue and/or closure of the relevant pull request.
+* Other decisions will be detailed as a narrative blog post.
```

Comment:
What about confidential / private matters? We should make this more flexible. The openness described above is a goal, but it should not be strictly required.